src/main/java/qna/domain/BaseEntity.java

```diff
@@ -3,19 +3,20 @@
 import java.time.LocalDateTime;
 
 import javax.persistence.Column;
+import javax.persistence.EntityListeners;
 import javax.persistence.MappedSuperclass;
 
-import org.hibernate.annotations.CreationTimestamp;
-import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
+@EntityListeners(AuditingEntityListener.class)
 @MappedSuperclass
 public class BaseEntity {
```

Comment:
```suggestion
public abstract class BaseEntity {
```

상속만을 위한 클래스라면 abstract를 통해 의도를 나타내보아요.